src/components/core/Icon/Icon.md

```diff
@@ -35,7 +35,7 @@ import Icon from '../Icon';
 import Link from '../Link';
 import Text from '../Text';
 <Text>
-  <Flex as={Link} href="/" alignItems="center">
+  <Flex as={Link} href="/" display="inline-flex" alignItems="center">
```

Comment:
Huh, I see now how you can override the `display` prop... disregard the comment on `Flex` then.